lib/base64.nit

```diff
@@ -33,14 +33,14 @@ redef class String
 		return inv_base64_chars
 	end
 
-	# Encodes the receiver string to base64.
-	# By default, uses "=" for padding.
-	fun encode_base64 : String do return encode_base64_custom_padding('='.ascii.to_b)
-
-	# Encodes the receiver string to base64 using a custom padding character.
+	# Encodes the received string to base64.
+	#
+	# If no padding is given, will use `=` as the padding character
 	#
-	# If using the default padding character `=`, see `encode_base64`.
-	fun encode_base64_custom_padding(padding : Byte) : String
+	#     assert "".encode_base64 == ""
+	#     assert "Morriar rly loves docs".encode_base64 == "TW9ycmlhciBybHkgbG92ZXMgZG9jcw=="
+	#     assert "Morriar rly loves docs".encode_base64(45.to_b) == "TW9ycmlhciBybHkgbG92ZXMgZG9jcw--"
```

Comment:
@R4PaSs no, `45u8` is as explicit as `45.to_b` unless when having the mapping table in front of me, I can't guess what char `45` represents.